### DISCUSSION TITLE
Issue with new installation using the LAN / ethernet

### DISCUSSION THREAD

**Discussion by richbayliss** [StealthChesnut/HA-FoxESS-Modbus#47] [Q&A] [answered]
I have hooked the `H1-3.7-E` up to an ethernet cable, and it acquires an IP from my gateway (192.168.35.184). After installing HACS and adding in this repo as-per the setup guide, upon restarting HASS I get these errors in the logs.

```
Logger: pymodbus.client.sync
Source: components/modbus/modbus.py:380
First occurred: 17:15:52 (209 occurrences)
Last logged: 17:21:12

Connection to (192.168.35.184, 502) failed: timed out
Connection to (192.168.35.184, 502) failed: [Errno 113] Host is unreachable
```

Is there a missing step, or what would be the best option to debug this? Sorry, I am very new to the FoxESS world 🤷🏻

**Comment by StealthChesnut** [comment, marked as answer]
This is more likely a basic network problem - your home assistant server can't connect over the network to the inverter. There's no network route between the two devices, hence "Host is unreachable" in your logs.

Can you ping both devices? Are they both on the same network segment? Are you using VLANs?
